### PR TITLE
Feature/Database connectivity

### PR DESCRIPTION
**Pull request by adrianshiokawa**
#2 

For testing:
- Create a 'pdh' Postgres database
- Create 'users' table
- In application, call dataHandling.Model.testFetch().toJSON()

### CODE REVIEW

**Comment by adrianshiokawa** [comment]
@felipesabino Ready for CR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# PostgreSQL Data Handling
+
+## Usage
+
+1. Set following environment variables:
+  - `DB_URL`: path for PostgreSQL database. Default: `localhost:5432`
+  - `DB_NAME`: DB name.
+  - `DB_USERNAME`: DB username. Default: `admin`
+  - `DB_PASSWORD`: DB password. Default: `admin`
+  - `MEMCACHE_SERVERS`: path for memcache. Default: `192.168.59.103:11211/memcache`
```
Comment (felipesabino) [inline README.md]: I think we can leave all `memcache` stuff on the side now, can't we?

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "postgres-data-handling",
+  "version": "0.1.0"
+}
```
Comment (felipesabino) [inline npm-shrinkwrap.json]: I believe all dependencies are missing here, may be you should have executed `npm shrinkwrap` before the commit/push

File: package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "homepage": "",
+  "dependencies": {
+    "async-timed-cargo": "0.0.3",
+    "bookshelf": "^0.9.2",
+    "coffee-script": "1.9.3",
+    "colors": "1.1.2",
+    "knex": "^0.10.0",
+    "lodash": "^4.0.0",
+    "memjs": "0.8.4",
+    "pg": "^4.5.1",
```
Comment (felipesabino) [inline package.json]: please fix version to avoid any further compatibility problems

File: src/helpers/config-helper.coffee [hunk 4]
```diff
@@ -0,0 +1,37 @@
+module.exports = (colors) ->
+
+  colors.setTheme
```
Comment (felipesabino) [inline src/helpers/config-helper.coffee]: all these methods ccan be included as a separate package https://github.com/tq1/br-tq1-helpers

File: coffeelint.json [hunk 5]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (felipesabino) [inline coffeelint.json]: isn't this `coffeelint.json` file duplicated?

**Comment by felipesabino** [comment]
So. I noticed it just later that you imported the old mongo project and is only doing modifications on top of it... Based on that you can ignore some of my comments regarding the old module, but in the future try to do separate PRs when this is the case, otherwise it is too hard to evaluate the changes properly

**Comment by adrianshiokawa** [comment]
The structural and setup changes suggested will be addressed on #3's PR

**Pull request merged**